app/src/main/java/helium314/keyboard/latin/Suggest.kt

```diff
@@ -115,13 +115,16 @@ class Suggest(private val mDictionaryFacilitator: DictionaryFacilitator) {
             inputStyleIfNotPrediction
         }
 
-        if (hasAutoCorrection) {
-            // make sure typed word is shown, so user is able to override incoming autocorrection
+        // If there is an incoming autocorrection, make sure typed word is shown, so user is able to override it.
+        // Otherwise, if the relevant setting is enabled, show the typed word in the middle.
+        val typedIndex = if (hasAutoCorrection) 2 else 1
```

Comment:
`typedIndex` is a confusing name, please be more explicit (e.g. `indexOfTypedWord`)